pkg/sdk/file_format.go

```diff
@@ -1,3 +1,4 @@
+//lint:file-ignore U1000 Ignore all unused code, it's generated
```

Comment:
is it really generated?

Reply:
no haha i just was getting annoyed by the linter showing me all the warnings of unused variables. i will remove it